website/project/model.py

```diff
@@ -615,6 +655,9 @@ class Node(GuidStoredObject, AddonModelMixin):
     # {<User.id>: [<Node._id>, <Node2._id>, ...] }
     child_node_subscriptions = fields.DictionaryField(default=dict)
 
+    # Files that contains comments
+    commented_files = fields.DictionaryField(default=dict)
```

Comment:
There're a lots of complications with the files comments. I talked to Jeff and he suggested doing so (maintaining a list of dict that holds all commented files)